traits/tests/test_view_elements.py

```diff
@@ -125,3 +125,8 @@ class Model(HasTraits):
 
         with self.assertRaises(TraitError):
             Model.class_trait_view_elements()
+
+    def test_view_element_superclass(self):
```

Comment:
Please add a `requires_traitsui` decorator to this test method, so that the test run still passes when TraitsUI is not installed into the development environment. It's available from the testing module:
```
from traits.testing.optional_dependencies import requires_traitsui, traitsui
```

Reply:
Sorry, ignore me. It's already there on the class. 🤦

Reply:
There is a `requires_traitsui` decorator on the whole class of tests. I think that should be enough?

Reply:
The whole test case class is decorated with `@requires_traitsui`, so I think it doesn't need this.